components/list/list-item-nav-button-mixin.js

```diff
@@ -25,19 +28,14 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 				width: 100%;
 			}
 			:host([current]) [slot="outside-control-container"] {
-				border: 3px solid var(--d2l-color-celestine);
+				background-color: var(--d2l-color-regolith);
+				border: 3px solid var(--d2l-color-celestine) !important;
```

Comment:
Without the `!important` mica was getting applied

Reply:
Is it a specificity issue? i.e. could this rule be made more specific so that it "wins"?

Reply:
The rule that's causing `mica` instead is very specific but I can try!